JavaTraining/BillsBurgers/src/Burger.java

```diff
@@ -0,0 +1,111 @@
+public class Burger {
+
+    protected String name;
+    protected double burgerPrice;
+    protected String breadType;
+    protected boolean meat;
+    protected boolean lettuce;
+    protected double lettucePrice = 0.60;
```

Comment:
Is there a possibility for delegation perhaps? Could we perhaps extract the add ons?